Runtime error panic with `gopm get`

This looks to me like a bug in the CLI processing arguments. Array/Slice is perhaps being reached into past its end.
```
gopm get -d racktop/brickstor/exec/subproc ssh://git@privaterepo.com/sz/exec.git -d
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/gpmgo/gopm/cmd.validPkgInfo(0x7fff5fbff345, 0x24, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/cmd/run.go:63 +0x536
github.com/gpmgo/gopm/cmd.getByPaths(0xc2080900b0, 0x0, 0x0)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/cmd/get.go:294 +0x2a0
github.com/gpmgo/gopm/cmd.runGet(0xc2080900b0)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/cmd/get.go:350 +0x442
github.com/gpmgo/gopm/modules/cli.Command.Run(0x459730, 0x3, 0x0, 0x0, 0x4d6250, 0x28, 0x529db0, 0x210, 0x0, 0x0, ...)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/modules/cli/command.go:101 +0xe42
github.com/gpmgo/gopm/modules/cli.(*App).Run(0xc20808c000, 0xc20800a000, 0x6, 0x6, 0x0, 0x0)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/modules/cli/app.go:125 +0xb70
github.com/gpmgo/gopm/lib.Run(0xc20800a000, 0x6, 0x6, 0x647190)
	/Users/szaydel/go/src/github.com/gpmgo/gopm/lib/lib.go:61 +0x6c0
main.main()
	/Users/szaydel/go/src/github.com/gpmgo/gopm/gopm.go:28 +0x51
```

I got same crash as ```gopm get -u -v gopkg.in/redis.v2-a@3fda679b48e6ba8cba04fb3a20a5d027d042c2``` on linux:

```
# gopm get -u -v gopkg.in/redis.v2-a@3fda679b48e6ba8cba04fb3a20a5d027d042c2
[GOPM] 15-09-18 10:16:20 [ INFO] App Version: 0.8.6.1228 Beta
[GOPM] 15-09-18 10:16:20 [ INFO] Local repository path: /root/.gopm/repos
[GOPM] 15-09-18 10:16:20 [ INFO] Indicated GOPATH: /mnt/gopkg
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/gpmgo/gopm/cmd.validPkgInfo(0x7fffd40b1d12, 0x26, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/mnt/gopkg/src/github.com/gpmgo/gopm/cmd/run.go:63 +0x520
github.com/gpmgo/gopm/cmd.getByPaths(0xc8200a6210, 0x0, 0x0)
	/mnt/gopkg/src/github.com/gpmgo/gopm/cmd/get.go:296 +0x329
github.com/gpmgo/gopm/cmd.runGet(0xc8200a6210)
	/mnt/gopkg/src/github.com/gpmgo/gopm/cmd/get.go:352 +0x4ca
github.com/gpmgo/gopm/modules/cli.Command.Run(0x8d1a28, 0x3, 0x0, 0x0, 0x94efc0, 0x28, 0x99c9e0, 0x210, 0x0, 0x0, ...)
	/mnt/gopkg/src/github.com/gpmgo/gopm/modules/cli/command.go:101 +0xdbc
github.com/gpmgo/gopm/modules/cli.(*App).Run(0xc8200fa000, 0xc82000a050, 0x5, 0x5, 0x0, 0x0)
	/mnt/gopkg/src/github.com/gpmgo/gopm/modules/cli/app.go:125 +0xa9f
github.com/gpmgo/gopm/lib.Run(0xc82000a050, 0x5, 0x5, 0x8474c0)
	/mnt/gopkg/src/github.com/gpmgo/gopm/lib/lib.go:61 +0x793
main.main()
	/mnt/gopkg/src/github.com/gpmgo/gopm/gopm.go:28 +0x60

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

# gopm -v
Gopm version 0.8.6.1228 Beta
```

test with new version in master branch

--- a/cmd/run.go
+++ b/cmd/run.go
@@ -59,10 +59,10 @@ func validPkgInfo(info string) (doc.RevisionType, string, error) {
 	}
 
 	infos := strings.Split(info, ":")
-	tp := doc.RevisionType(infos[0])
-	val := infos[1]
 
 	if len(infos) == 2 {
+		tp := doc.RevisionType(infos[0])
+		val := infos[1]
 		switch tp {
 		case doc.BRANCH, doc.COMMIT, doc.TAG:
 		default:

